### ISSUE TITLE
OID equivalents of UUID() and STRUUID()

### ISSUE THREAD

**Issue by ebremer** [w3c/sparql-dev#204]
## Why?

Some standards and systems like to use [OIDs](http://www.oid-info.com/index.htm#oid) such as [DICOM](https://dicom.nema.org/medical/dicom/current/output/chtml/part05/chapter_9.html)

The OID form of a UUID is shown [here](http://www.oid-info.com/cgi-bin/display?oid=2.25&action=display)
Where the hexidecimal UUID is appended to "2.25." in decimal (base 10) form with nothing but numbers (no dashes).

## Previous work

Similar to UUID() and STRUUID()

## Proposed solution
Take UUID "f81d4fae-7dec-11d0-a765-00a0c91e6bf6"

In OID world would be:
```
OIDUUID() -> "urn:oid:2.25.329800735698586629295641978511506172918"

STROIDUUID() -> "2.25.329800735698586629295641978511506172918"
```

**Comment by larsw** [comment]
Interesting, but shouldn't the `OID()` work on ASN.1 object identifiers (OIDs) in general?

**Comment by ebremer** [comment]
@larsw no, UUID strings are in hexidecimal and need to be converted to decimal before being prefixed with "2.25." for a proper OID-UUID.   But I like the thought you brought up, a general oid() function which takes a valid oid string and returns urn:oid:<oid string> could be useful.  It wouldn't work for oid(struuid()) since the current SPARQL struuid() would return a hexidecimal UUID instead of a decimal UUID.
